### ISSUE TITLE
[L4D2] New type of cheating

### ISSUE THREAD

**Issue by ProjectSky** [ValveSoftware/Source-1-Games#5178]
i know there are many types of cheats, but this one is special and completely breaks the balance of vs mode.
It can be seen from the video that the charger cannot catch the player, but not only the charger, all control si cannot catch this player

https://github.com/ValveSoftware/Source-1-Games/assets/16351827/9bb7e51c-3be2-49cb-b135-c271f1a2cd2c

**Comment by eyal282** [comment]
Did you verify, assuming you can properly do that, that the server owner did not protect the given player from charges, as this is a community server, and myself have made immunity to charger when you activate something in !buy

However I will note, that the "Grab" sound is heard when you use Left 4 Dhooks in Sourcemod to block. However, you can also in theory block that sound.

**Comment by ProjectSky** [comment]
> Did you verify, assuming you can properly do that, that the server owner did not protect the given player from charges, as this is a community server, and myself have made immunity to charger when you activate something in !buy
> 
> However I will note, that the "Grab" sound is heard when you use Left 4 Dhooks in Sourcemod to block. However, you can also in theory block that sound.

This is my own server and players have reported this type of cheating many times

**Comment by NULLYUKI** [comment]
This seems more like a issue related to the network connection.
I searched if the issue was already reported somewhere, and I found a video on Reddit which exactly shows the same issue.
Here is the reddit post [Charger detection in action.](https://www.reddit.com/r/l4d2/comments/tx9qy3/charger_detection_in_action/?rdt=49183)
In the reddit post, a explanation about the issue can be found.
![grafik](https://github.com/ValveSoftware/Source-1-Games/assets/42326194/eb811a17-0f59-4836-938f-0a51c3c533a7)

That means, that the issue is most likely caused by the ping of the player and their Network settings in the game.
You can read here about the explanation of [Lag compensation](https://developer.valvesoftware.com/wiki/Source_Multiplayer_Networking#Lag_compensation)

**Comment by ProjectSky** [comment]
> This seems more like a issue related to the network connection. I searched if the issue was already reported somewhere, and I found a video on Reddit which exactly shows the same issue. Here is the reddit post [Charger detection in action.](https://www.reddit.com/r/l4d2/comments/tx9qy3/charger_detection_in_action/?rdt=49183) In the reddit post, a explanation about the issue can be found in the comments. ![grafik](https://user-images.githubusercontent.com/42326194/265635817-eb811a17-0f59-4836-938f-0a51c3c533a7.png)
> 
> That means, that the issue is most likely caused by the ping of the player and their Network settings in the game. You can read here about the explanation of [Lag compensation](https://developer.valvesoftware.com/wiki/Source_Multiplayer_Networking#Lag_compensation)

I don't think so, if it's a ping/network issue, there should have been similar reports before, but it never did. And I've observed multiple target players and they basically have VAC bans

not only charger all control SI can't catch this player

**Comment by NULLYUKI** [comment]
> > This seems more like a issue related to the network connection. I searched if the issue was already reported somewhere, and I found a video on Reddit which exactly shows the same issue. Here is the reddit post [Charger detection in action.](https://www.reddit.com/r/l4d2/comments/tx9qy3/charger_detection_in_action/?rdt=49183) In the reddit post, a explanation about the issue can be found in the comments. ![grafik](https://user-images.githubusercontent.com/42326194/265635817-eb811a17-0f59-4836-938f-0a51c3c533a7.png)
> > That means, that the issue is most likely caused by the ping of the player and their Network settings in the game. You can read here about the explanation of [Lag compensation](https://developer.valvesoftware.com/wiki/Source_Multiplayer_Networking#Lag_compensation)
> 
> I don't think so, if it's a ping/network issue, there should have been similar reports before, but it never did. And I've observed multiple target players and they basically have VAC bans
> 
> not only charger all control SI can't catch this player

I found another report on reddit of that kinda issue with the charger.
[Charger bug?](https://www.reddit.com/r/l4d2/comments/9ur6i2/charger_bug/)

I would suggest trying out to tweak your network configuration or from your server and look if the issue is now less occurring.

**Comment by ProjectSky** [comment]
> > not only charger all control SI can't catch this player
> 
> Could we see more examples of this? I have had verified Official Dedicated games where I'd pull the player up to me as Smoker and start scratching but they'd take no damage; so it could be similar, and is viable as a cheat. The Charger video is not conclusive of anything, though.

I will provide more video or demo, it will take some time

**Comment by ProjectSky** [comment]
* new video

https://github.com/ValveSoftware/Source-1-Games/assets/16351827/b2cf87d3-164a-473c-befe-38e72adbbb9d

**Comment by NULLYUKI** [comment]
It would be good to know what network configuration the player had and a video from their view with `net_graph 1` displayed.

By just entering a network command the console will printout the set value, so following network configurations would be good to have:
```
cl_cmdrate
cl_updaterate
cl_interp_ratio
cl_lagcompensation
cl_interp
```
